widgetssdk/src/testSnapshot/java/com/glia/widgets/entrywidget/EntryWidgetEmbeddedViewTest.kt

```diff
@@ -72,6 +79,51 @@ internal open class EntryWidgetEmbeddedViewTest : SnapshotTest(
         )
     }
 
+    // MARK: Contacts Tests with unread count badge
+
+    @Test
+    fun contactsWithBadgeDefaultTheme() {
+        snapshot(
+            setupView(
+                items = contactsItems,
+                observeUnreadMessagesCountUseCase = observeUnreadMessagesCountFiveUseCaseMock,
```

Comment:
Why not pass the number of messages to the `setupView()` method instead of the mock?
In this case, you can create the mock of `ObserveUnreadMessagesCountUseCase` inside the `setupView()`.

Reply:
It's no longer the case after this commit 619d518fac903575fde6e24f9cbb9e471c6fc81a 😉